.github/workflows/CI.yml

```diff
@@ -130,3 +130,12 @@ jobs:
         run: |
           mkdir -p /home/runner/.ivy2/cache/ # some scripts assume that this folder exists
           ${{ matrix.script }}
+
+
+      - uses: actions/checkout@master
+      - uses: codecov/codecov-action@v2
+        with:
+          # files: ./coverage1.xml,./coverage2.xml # optional
+          name: codecov
+          fail_ci_if_error: true
+          verbose: true
```

Comment:
the documentation assures that no credentials are needed using the provided action. I'm not sure how the uploaded works and if it can auto-detect the generated coverage files